catboost/python-package/catboost/widget/metrics_plotter.py

```diff
@@ -0,0 +1,218 @@
+import time
+import logging
+
+import typing as tp
+
+from traitlets import Unicode, Dict, default
+from IPython.display import display
+from ipywidgets import DOMWidget, Layout, widget_serialization
+from copy import deepcopy
+from typing import List, Optional, Any, Union
+
+logging.getLogger().setLevel(logging.DEBUG)
+
+class MetricsWidget(DOMWidget):
```

Comment:
Maybe use common base class for `MetricsWidget` and `MetricVisualizer` to store repeated defs?